project/TestGroups.scala

```diff
@@ -56,7 +61,27 @@ object TestGroups {
       "tests.MessagesSuite", "tests.pants.PantsGlobsSuite",
       "tests.SelectBspServerSuite", "tests.InverseDependenciesSuite",
       "tests.pantsbuild.PantsProjectNameSuite", "tests.TrigramSubstringsSuite",
```

Comment:
It seems that some of tests were removed. 
Like: `tests.pantsbuild.PantsProjectNameSuite`

Reply:
Yeah, I need try also a warning for tests that were removed, but not sure how to do it aside from manual.

It doesn't actually cause any issues though.